### DISCUSSION TITLE
15페이지부터 실행되지 않아서 여쭤봅니다!

### DISCUSSION THREAD

**Discussion by yerinP** [Road-Balance/gcamp_ros2_basic#21] [Q&A]
wsl 버전 2로 변경하는 명령어를 실행하면, 
![image](https://user-images.githubusercontent.com/109068797/178227707-21f52174-f4b2-4793-8e81-b78a61ec822c.png)

이렇게 뜨는데 어떻게 해결해야하나요??
WSL 2 Linux 커널 업데이트 설치/진행 후 재부팅 하고, PowerShell 도 관리자권한으로 실행했습니다..

**Comment by kimsooyoung** [comment]
오류 내용으로 짐작하건데 명령어에 오타나 띄어쓰기 오류가 있어 보입니다.

```
wsl --set-default-version 
```

명령어를 직접 손으로 타이핑해보세요

**Reply by yerinP** [reply]
그대로 붙여넣어도 같은 오류가 뜹니다ㅠ

**Reply by kimsooyoung** [reply]
```
wsl --install
```

위 명령어 실행 이후 결과를 알려주세요

**Reply by yerinP** [reply]
![image](https://user-images.githubusercontent.com/109068797/178404373-15a560ab-b19d-4811-889b-9a65a1422f16.png)
 같은 결과가 나옵니다!
 검색으로 윈도우 버전 20이상이어야 한다는 사실을 알게되었는데요, 지금 제 컴퓨터는 업데이트중이라고 뜨면서 어제부터 업데이트가 멈춰있는 상태입니다!

**Reply by kimsooyoung** [reply]
업데이트는 잘 완료되셨나요? 업데이트 이후 다시 첫 PPT부터 시도해보시며 오류 결과를 공유해주세요!

**Reply by yerinP** [reply]
업데이트가 이틀 째 완료가 되지 않아서 아직 시도해보지 못했습니다,, 완료되고 다시 알려드리겠습니다!

**Reply by kimsooyoung** [reply]
수업은 우선 docker 환경에서 따라오신 뒤 이후 다시 설정해보세요. 나중에도 이곳에 질문 올려주시면 받겠습니다.
